blockware/matrix/lib/Touch/Touch.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+// Accelerometer
+#include <LIS2DW12Sensor.h>
+
+class Touch {
+private:
+  static const uint8_t axes_count = 3;
+  static const uint8_t axes_history_length = 15;
+  static const int16_t DEFAULT_SENSITIVITY = 25;
```

Comment:
nit: Why all caps on this one?

Reply:
good point i could change all const to be caps i sort of mixed up doing statics and consts in header files and in the cpp files themselves after i discovered both work (didn’t realize we could initialize in header file but it makes sense)